src/main/resources/db/postgres/V1_89__vaas.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE VAAS
+(
+    hendelse_id      varchar(20)  NOT NULL PRIMARY KEY,
+    ident            VARCHAR(20)  NOT NULL,
```

Comment:
kunne vi enten satt fnr eller 11 på varchar? synes det er så stress å måtte logge inn i db og måtte hente ut rader for å skjønne hva vi bruker på personen 🙈